modular_zubbers/code/game/machinery/computer/arcade/arcade_weights_oh_god.dm

```diff
@@ -0,0 +1,12 @@
+GLOBAL_LIST_INIT(arcade_prize_pool_oh_god, list(
+	/obj/item/card/emag/one_shot = 1,
+	/obj/item/clothing/shoes/clown_shoes/banana_shoes = 1,
+	/obj/item/clothing/shoes/gunboots/disabler = 1,
```

Comment:
See what I said before about this, but I grabbed official opinions from the staff team and it's a hard no, unfortunately. You're going to have to find better items for a 1/5000 chance and ones that aren't going to incentivize extremely poor the moment someone gets them. Think a security officer chasing someone down the hall with these boots on.